mw-webapp/cypress/scopesSelectors/headerSelectors.ts

```diff
@@ -0,0 +1,9 @@
+import { headerAccessIds } from "cypress/accessIds/headerAccessIds";
+import { getDataCy } from "src/utils/cyTesting/getDataCy";
+
+export const headerSelectors = {
+    items: {
+        getLogoBtn: () => cy.get(getDataCy(headerAccessIds.items.allWaysLogoBtn)),

```

Comment:
I think there it will be nice to have  direct connection between id an selector, we can achieve it by just changing prefixes: 
for example:
id:  allWaysLogoBtn
selector: getAllWaysLogoBtn


i think it will be easier to work with such system in the future, what do you think?